README.md

```diff
@@ -1,27 +1,55 @@
-# privacy-evaluator
+# Privacy Evaluator
+<!--[![Gitter](https://badges.gitter.im/fairlearn/community.svg)](https://gitter.im/fairlearn/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)-->
+
+- space for the description of our project (core idea)
+
+- For example:
+The privML Privacy Evaluator is a tool that assesses ML model's levels of privacy by running different attacks on it. .....
+
+## Dependencies
+- like necessary packages
+- version of packages
+- we have to keep this section up-to-date during development
 [![Python 3.7](https://img.shields.io/badge/python-3.7-blue.svg)](https://www.python.org/downloads/release/python-370/)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
-[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
-[![Gitter](https://badges.gitter.im/fairlearn/community.svg)](https://gitter.im/fairlearn/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-The privML Privacy Evaluator is a tool that assesses ML model's levels of privacy by running different attacks on it.
-
-## How does it work?
+## Installation
+- if we need this section
+- could be a command line code
 
 ## How to use the Privacy Evaluator?
+- description how our project works
 
-## How to contribute to the project?
-If you want to contribute in any way, please visit our [Contribution Guidelines](https://github.com/privML/privacy-evaluator/CONTRIBUTING.md) to get started.
 
-## Communication
+### Example
+- some examples would be good
 
-### Who is behind the project?
-See a list of our [Supporters]().
+
+## Attacks
+
+### Property Inference Attack
+
+### Membership Inference Attack
+
+
+## Getting Involved
+If you want to contribute in any way, please visit our [Contribution Guidelines](./CONTRIBUTING.md) to get started. Please have also a look at our [Code of Conduct](./CODE_OF_CONDUCT.md). 
 
 ### Contact
 Gitter: https://gitter.im/privML/community
+You can join our [Gitter](https://gitter.im/privML/community) communication channel.
+
+### Who is behind the project?
```

Comment:
Please link the list of supporters here or just write down the 3 supporters that I told you about.

Reply:
The relevant version of the README can be found in the pull request "Update of README.md #32". This pull request is only concerning PIA interface and not up to date on the README.